sdk/tables/data-tables/README.md

```diff
@@ -223,6 +226,34 @@ async function main() {
 main();
 ```
 
+Here is a sample that checks if the table already existed
+
+```javascript
+const { TableServiceClient, AzureNamedKeyCredential } = require("@azure/data-tables");
+
+const account = "<account>";
+const accountKey = "<accountkey>";
+
+const credential = new AzureNamedKeyCredential(account, accountKey);
+const serviceClient = new TableServiceClient(
+  `https://${account}.table.core.windows.net`,
+  credential
+);
+
+async function main() {
+  const tableName = `newtable${new Date().getTime()}`;
```

Comment:
should we get rid of the date/gettime so that it can actually already exist the second time they run it and maybe put a comment to that effect?